contrib/vagrant/provision-util.sh

```diff
@@ -81,6 +81,9 @@ os::provision::init-certs() {
   local volumes_dir="/var/lib/openshift.local.volumes"
   local cert_dir="${server_config_dir}/master"
 
+  # Clear stale configuration
+  rm -rf ${config_root}/openshift.local.*
```

Comment:
I realize the confusion around the intent of this PR is due to my conflating 2 separate issues:

 - enabling parallel node deployment
 - removing an existing etcd directory because it breaks deployment when mounted over nfs

I'll retarget this PR to target only the first issue, and submit a second PR to target the second by removing the etcd directory only when nfs is configured.